src/script/components/list/ConversationListCell.tsx

```diff
@@ -151,11 +151,58 @@ const ConversationListCell = ({
     }
   }, [isFocused]);
 
-  const availabilityStrings: Record<string, string> = {
-    [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
-    [Availability.Type.AWAY]: t('userAvailabilityAway'),
-    [Availability.Type.BUSY]: t('userAvailabilityBusy'),
-  };
+  const availabilityStrings: Record<string, string> = useMemo(
+    () => ({
+      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
+      [Availability.Type.AWAY]: t('userAvailabilityAway'),
+      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
+    }),
+    [],
+  );
```

Comment:
```suggestion
  const availabilityStrings = useMemo(
    () => ({
      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
      [Availability.Type.AWAY]: t('userAvailabilityAway'),
      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
    } as const),
    [],
  );
```

That should work, right?